test/Microsoft.TestPlatform.Extensions.BlameDataCollector.UnitTests/XmlReaderWriterTests.cs

```diff
@@ -120,7 +121,8 @@ public void WriteTestSequenceShouldWriteFileStream()
     {
         // Setup
         _mockFileHelper.Setup(m => m.Exists(It.IsAny<string>())).Returns(true);
-        _mockFileHelper.Setup(m => m.GetStream("path.xml", FileMode.Create, FileAccess.ReadWrite)).Returns(_mockStream.Object);
```

Comment:
Changed this because mock is trying to create Span and fails. It is a known limitation. So I just use real stream.